src/cascadia/UnitTests_TerminalCore/ConptyRoundtripTests.cpp

```diff
@@ -673,6 +677,230 @@ void ConptyRoundtripTests::MoveCursorAtEOL()
     verifyData1(termTb);
 }
 
+void ConptyRoundtripTests::TestResizeHeight()
+{
+    // This test class is _60_ tests to ensure that resizing the terminal works

```

Comment:
This reminds me of
![](https://images-na.ssl-images-amazon.com/images/I/81CuYjT8DhL._AC_SL1500_.jpg)